src/components/Description/Description.jsx

```diff
@@ -1,11 +1,11 @@
 import { Button } from '../Button/Button';
-import './Description.css';
+import styles from './Description.module.css';
 
 export const Description = () => {
     return (
-        <section className="description">
-            <h1 className="description-header">Projects</h1>
-            <p className="description-text">
+        <section className={styles.description}>
+            <h1 className={styles['description-header']}>Projects</h1>
+            <p className={styles['description-text']}>
```

Comment:
As you can see `-`ed css classes are not convenient to use with css modules. Feel free to switch to camelCase or snake_case for the class names in the future tasks. 
No need to update anything now
Furthermore you can stop using css methodologies for class naming since css modules solve the collision problem. Just make sure they still have meaningful names.